packages/qwik-city/middleware/node/http.ts

```diff
@@ -89,13 +89,18 @@ export async function fromNodeHttp(
     },
     getWritableStream: (status, headers, cookies) => {
       res.statusCode = status;
-      headers.forEach((value, key) => res.setHeader(key, value));
-      const cookieHeaders = cookies.headers();
-      if (cookieHeaders.length > 0) {
-        res.setHeader('Set-Cookie', cookieHeaders);
-      }
+      let once = false;
```

Comment:
This is an oversight I guess

Reply:
I want to write once since write can happen multiple times (unless it's renderToString)

Reply:
I see. I think this should be true to execute the next if once

Reply:
oh yeah I meant to set the other one to true